### PR TITLE
[docker-orchagent]: Generate config-files for IPinIP and Everflow.

### PR DESCRIPTION
**Pull request by antonpatenko**
* Added templates for IPinIP and Everflow configuration file.
    * Added config.sh to generate configs using templates.
    * Extended minigraph.py to read ErspanIPv4 section.

Use {{ erspan_dst[0] }} to read Everflow destination IPv4 address in your config/minigraph file.

### CODE REVIEW

File: dockers/docker-orchagent/ipinip.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[
+    {
+        "TUNNEL_DECAP_TABLE:IPINIP_TUNNEL" : {
+            "tunnel_type":"IPINIP",
+            "src_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dst_ip":"{{ minigraph_lo_interfaces[0]['addr'] }}",
+            "dscp_mode":"uniform",
+            "ecn_mode":"standard",
```
Comment (lguohan) [inline dockers/docker-orchagent/ipinip.conf.j2]: by default this should be copy_from_outer, can you check the asic_type and generate corresponding configuration based on the asic_type?

Comment (antonpatenko) [inline dockers/docker-orchagent/ipinip.conf.j2]: @lguohan 
which exactly field should be copy_from_outer?

File: dockers/docker-orchagent/config.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -e
+
```
Comment (stcheng) [inline dockers/docker-orchagent/config.sh]: check if the folder exists. later swss.deb package might change.

File: dockers/docker-orchagent/start.sh [hunk 3]
```diff
@@ -50,7 +50,7 @@ elif [ "$HWSKU" == "AS7512" ]; then
 elif [ "$HWSKU" == "INGRASYS-S9100-C32" ]; then
     ORCHAGENT_ARGS+="-m $MAC_ADDRESS"
 elif [ "$HWSKU" == "ACS-MSN2700" ]; then
-    SWSSCONFIG_ARGS+="msn2700.32ports.buffers.json msn2700.32ports.qos.json "
+    SWSSCONFIG_ARGS+="msn2700.32ports.buffers.json msn2700.32ports.qos.json ipinip.conf mirror.conf "
```
Comment (taoyl-ms) [inline dockers/docker-orchagent/start.sh]: Should it be limited to MLNX platform?

File: dockers/docker-orchagent/ipinip.conf.j2 (hunk 1 above)
Comment (lguohan) [inline dockers/docker-orchagent/ipinip.conf.j2]: the ecn_mode

**Pull request closed without merging**